packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -28,13 +28,13 @@ import BackupIntroduction from 'src/backup/BackupIntroduction'
 import ExchangeHomeScreen from 'src/exchange/ExchangeHomeScreen'
 import WalletHome from 'src/home/WalletHome'
 import { Namespaces } from 'src/i18n'
-import AccountKey from 'src/icons/navigator/AccountKey'
-import AddWithdraw from 'src/icons/navigator/AddWithdraw'
-import Gold from 'src/icons/navigator/Gold'
-import Help from 'src/icons/navigator/Help'
-import Home from 'src/icons/navigator/Home'
-import Invite from 'src/icons/navigator/Invite'
-import Settings from 'src/icons/navigator/Settings'
+import { AccountKey } from 'src/icons/navigator/AccountKey'
```

Comment:
why import the non memorized version?

Reply:
Yeah, think you may need to drop the destructuring but not 100% sure